### PR TITLE
release note for aws lambda

### PR DESCRIPTION
**Pull request by nmohantyctr**
## Purpose of this pull request

This pull request (PR) ...

Issue number: 

## Select the type of change:

- [ ] Minor Changes - Typos, formatting, slight revisions
- [ ] Update Content - Revisions and updating sections
- [x] New Content - New features, sections, pages, tutorials
- [ ] Site and Tools - Updates, maintenance, and new packages for the site, Gatsby, React, etc

### CODE REVIEW

File: blog-service/2022-11-21-apps [hunk 1]
```diff
@@ -0,0 +1,17 @@
...
+  - apps
+hide_table_of_contents: true
+authors:
+  - url: https://help.sumologic.com/release-notes-service/rss.xml
+    image_url: /img/release-notes/rss-orange.png
+---
+
+import useBaseUrl from '@docusaurus/useBaseUrl';
+
+We are happy to announce that we have released an updated version of [Sumo Logic Lambda Extension](https://github.com/SumoLogic/sumologic-lambda-extensions) that supports collecting the telemetry data using the new [AWS Lambda Telemetry API](https://docs.aws.amazon.com/lambda/latest/dg/telemetry-api.html). The extension gives a per-invoke level view of the Lambda function and provides insights into the function and execution environment lifecycle. With this update, you can collect telemetry data including logs, metrics, and traces in JSON format directly from the Lambda execution level. For more information on how to start using it please refer to our [documentation](https://help.sumologic.com/docs/send-data/collect-from-other-data-sources/collect-aws-lambda-logs-extension/).
```
Comment (JainM6) [inline blog-service/2022-11-21-apps:17]: I'd rephrase this sentence as - "We are pleased to announce that the Sumo Logic Lambda Extension has been updated to support collecting telemetry data using the new AWS Lambda Telemetry API".

Comment (JainM6) [inline blog-service/2022-11-21-apps:17]: https://github.com/SumoLogic/sumologic-lambda-extensions. Will this be available to external customers?

Comment (JainM6) [inline blog-service/2022-11-21-apps:17]: The extension gives a per-invoke level view of the Lambda function and provides insights into the function and execution environment lifecycle. 
Suggestion: I didn't understand the word "per-invoke", I think it should be pre-invoke or pre-invocation. Could you please check this with SME again?

Comment (JainM6) [inline blog-service/2022-11-21-apps:17]: For more information on how to start using it please refer to our documentation.
Suggestion: We should avoid using "please" in the user documentation.

**Review by JainM6 — CHANGES_REQUESTED** [review]

File: blog-service/2022-11-21-apps (hunk 1 above)
Comment (nmohantyctr) [reply blog-service/2022-11-21-apps:17]: Yes

Comment (nmohantyctr) [reply blog-service/2022-11-21-apps:17]: Just verified with the SME it is meant for 'per-invoke'. Here is the slack message:

"The idea is that each time the Lambda function is invoked we get the data immediately."

Comment (nmohantyctr) [reply blog-service/2022-11-21-apps]: Have made the changes.

**Review by JainM6 — APPROVED** [review]
Thanks, Nishant. This looks good to me. I have just removed "please".
